static/app/views/dashboards/widgetBuilder/components/newWidgetBuilder.spec.tsx

```diff
@@ -286,4 +295,48 @@ describe('NewWidgetBuiler', function () {
       expect(screen.queryByLabelText('Remove this filter')).not.toBeInTheDocument();
     });
   });
+
+  it('renders the group by field on chart widgets and can function', async function () {
```

Comment:
Does this test need to be on the new widget builder? It seems to only assert on the field itself, I think we can make a specific `groupBySelector.spec.tsx` for this so we can start breaking up what gets rendered in tests

Reply:
If you wanted a widget builder test, I think testing that logic with the `isChartWidget` logic is worth putting here. e.g. just check that when `displayType: 'line'`, the group by field is shown

Reply:
ah i see, yup i can make a `groupBySelector.spec.tsx` file